src/templates/views/payment.hbs

```diff
@@ -41,35 +41,38 @@
 
             <h2>Shipping address</h2>
 
-            <input id="shipping-same" name="shipping-shipping-same" type="checkbox">
-            <label for="shipping-shipping-same">Same as billing address</label>
+            <details>
+                <summary>Enter shipping address (optional)</summary>
 
-            <label for="shipping-honorific-prefix">Title</label>
-            <input required autocomplete="honorific-prefix" id="shipping-honorific-prefix" name="shipping-honorific-prefix">
+                <input id="shipping-same" name="shipping-shipping-same" type="checkbox">
+                <label for="shipping-shipping-same">Same as billing address</label>
```

Comment:
Optional: Change summary to Use different shipping address and get rid of this checkbox.

Reply:
Taken care of in https://github.com/GoogleChromeLabs/sample-pie-shop/pull/109/commits/7f31b064316fe603272ca0df897b3b1ce6e6848a.